src/style.css

```
@@ -0,0 +1,129 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+}
+
+.bigForm {
+  position: relative;
+  margin: 20px 0;
+}
+
+.labelBig {
+  font-weight: 300;
+}
+
+.big {
+  height: 70px;
+  font: inherit;
+  box-sizing: border-box;
```

```suggestion
  font-family: inherit;
```
remove `box-sizing` from everything, you've already applied it to `*`